packages/core/src/rum/instrumentation/DdRumUserInteractionTracking.tsx

```diff
@@ -54,6 +54,9 @@ export class DdRumUserInteractionTracking {
         const originalMemo = React.memo;
         React.memo = (component: any, propsAreEqual?: (prevProps: any, newProps: any) => boolean) => {
             return originalMemo(component, (prev, next) => {
+                if (!next.onPress) {
```

Comment:
can you explain a bit here, why are you not checking also the `prev.onPress` ?

Reply:
Doing the object spread can cost a bit of performance if you run `memo` on a large amount of components (it can easily be 100 components) - so if some of them don't have an onPress prop we can skip this step and have a lighter impact.

I can definitely change it to `(!next.onPress || !prev.onPress)` as there is no reason to check only one.